src/CFixer/main.cpp

```diff
@@ -7,45 +7,24 @@
 
 #include <quasarapp.h>
 #include <iostream>
+#include <CopyrighFixer/worker.h>
+
+using namespace CopyrighFixer;
 
-void helpCall();
 
 int main(int argc, char *argv[]) {
 
+    Worker *worker = new Worker();
```

Comment:
you create the worker object on heap but not free it. Use the stack memory for automaticity clearing.
```suggestion
    Worker worker;
```